Libs/DICOM/Widgets/Plugins/CMakeLists.txt

```diff
@@ -35,6 +31,19 @@ set(PLUGIN_MOC_SRCS
 set(PLUGIN_resources
 )
 
+if(CTK_QT_VERSION VERSION_GREATER "4")
```

Comment:
The check should be done based on enabled/disabled features. We should avoid features disappearing silently based on detected Qt version.